### PR TITLE
Bug fixes to media gallery

### PR DESCRIPTION
**Pull request by Omonroy36**

### CODE REVIEW

File: src/app/services/breathecode.js [hunk 1]
```diff
@@ -175,12 +175,15 @@ class BreatheCodeClient {
...
       },
       getAcademyMembers: (query) => {
-        const qs = Object.keys(query)
-          .map((key) => `${key}=${query[key]}`)
-          .join("&");
+        const qs =
+          query !== undefined
+            ? Object.keys(query)
+                .map((key) => `${key}=${query[key]}`)
+                .join("&")
```
Comment (ArielCalisaya) [inline src/app/services/breathecode.js:182]: change js quotes elements to `single-quote`

File: src/app/views/admissions/student-form/NewStudent.jsx [hunk 2]
```diff
@@ -4,21 +4,26 @@ import { Card, Divider, Button } from '@material-ui/core';
 import { Breadcrumb } from 'matx';
 import { createFilterOptions } from '@material-ui/lab/Autocomplete';
 import { AsyncAutocomplete } from 'app/components/Autocomplete';
+import { useQuery } from '../../../hooks/useQuery';
 import bc from 'app/services/breathecode';
 import { ProfileForm } from './student-utils/ProfileForm';
-
+//atob
```
Comment (ArielCalisaya) [inline src/app/views/admissions/student-form/NewStudent.jsx:10]: remove unused comments

File: src/app/views/admissions/student-form/NewStudent.jsx [hunk 3]
```diff
@@ -4,21 +4,26 @@ import { Card, Divider, Button } from '@material-ui/core';
...
 const filter = createFilterOptions();
 const NewStudent = () => {
+  const query = useQuery();
+  const baseData = query.has("data") ? JSON.parse(atob(query.get("data"))) : null;
```
Comment (ArielCalisaya) [inline src/app/views/admissions/student-form/NewStudent.jsx:14]: use `single-quotes` in javascript elements

File: src/app/views/events/events.jsx [hunk 4]
```diff
@@ -1,18 +1,14 @@
-import React, { useState, useEffect } from 'react';
-import { Breadcrumb, MatxLoading } from 'matx';
-import MUIDataTable from 'mui-datatables';
+import React, { useState } from 'react';
+import { Breadcrumb } from 'matx';
+import { SmartMUIDataTable } from 'app/components/SmartDataTable';
```
Comment (ArielCalisaya) [inline src/app/views/events/events.jsx:3]: ESlint expects relative import like `../../../components/etc`

File: src/app/views/admissions/students.jsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import { Breadcrumb, MatxLoading } from 'matx';
+import { Breadcrumb } from 'matx';
```
Comment (ArielCalisaya) [inline src/app/views/admissions/students.jsx:2]: ESlint expects relative import like `../../../components/etc`

File: src/app/views/feedback/answers.jsx [hunk 6]
```diff
@@ -1,13 +1,10 @@
 /* eslint-disable no-nested-ternary */
-import React, { useState, useEffect, Fragment } from 'react';
-import { Breadcrumb, MatxLoading, MatxMenu } from 'matx';
-import MUIDataTable from 'mui-datatables';
+import React, { useState } from 'react';
+import { Breadcrumb } from 'matx';
```
Comment (ArielCalisaya) [inline src/app/views/feedback/answers.jsx:3]: ESlint expects relative import like `../../../components/etc`

File: src/app/views/leads/leads.jsx [hunk 7]
```diff
@@ -1,17 +1,12 @@
-import React, { useState, useEffect } from 'react';
-import {
-  Grow, Icon, IconButton, TextField, Button,
-} from '@material-ui/core';
+import React, { useState } from 'react';
+import { Button} from '@material-ui/core';
+import { SmartMUIDataTable } from 'app/components/SmartDataTable';
```
Comment (ArielCalisaya) [inline src/app/views/leads/leads.jsx:3]: ESlint expects relative import like `../../../components/etc`

**Review by ArielCalisaya — CHANGES_REQUESTED** [review]

**Review by ArielCalisaya — APPROVED** [review]

**Review by alesanchezr — APPROVED** [review]

**Review by ArielCalisaya — APPROVED** [review]

**Pull request merged**